docs/getting-started.md

```diff
@@ -50,6 +50,12 @@ You will need to be running Linux, macOS, or Windows, and have [Python](https://
     pipx install meltano
     ```
 
+    If you have multiple versions of Python installed, you can use a specific one with the `--python` arugment:
+    
+    ```bash
+    pipx install meltano --python [path to desired Python exe]
```

Comment:
- `<...>` instead of `[...]` to indicate that the argument is not optional.
- `executable` instead of `exe` to make it clear that we aren't necessarily referring to a Windows `.exe` file.

```suggestion
    pipx install meltano --python <path to desired Python executable>
```